config/locales/en.yml

```diff
@@ -57,6 +57,7 @@ en:
   add_category: 'Add category'
   add_sub_category: 'Add sub category'
   subcategories: 'Subcategories'
+  can_not_be_removed: 'can not be removed'
```

Comment:
If it's possible, you can provide more informative message, like 'Unfortunately, it cannot be removed'